examples/windows-instance/variables.tf

```diff
@@ -22,24 +22,12 @@ variable "create_ec2_kms_key" {
   default     = true
 }
 
-variable "create_key_pair" {
```

Comment:
Hi Patrick, How do we connect to the windows instance created?

Reply:
@ndegwajohn through ssm. Although I may need to enable password connection for the windows instance by allowing one to provide a key pair name

Reply:
Okay. 
The ssm agent takes abit of time to install in windows. Now I've seen the option to connect.
After connecting I see a PowerShell terminal but I am not able to interact with it, is it the same case in your end?

Reply:
@hugoalmeida264 as of now there is now the viable options to connect to the windows instance is by use of keypair to generate the password, or using Instance connect which also requires the use of key pair